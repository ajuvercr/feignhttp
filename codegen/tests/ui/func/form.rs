use feignhttp_codegen::post;

// error: one form parameter only supports scalar types, &str, String or struct
//    |  pub async fn send_post(#[form] s: &String) -> Result<String> {}
//    |                                 ^^^^^^^^^^

#[post("http://xxx")]
pub async fn send_post(#[form] s: &String) -> Result<String> {}

fn main() {}
