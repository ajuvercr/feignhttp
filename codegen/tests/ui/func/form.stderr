error: one form parameter only supports scalar types, &str, String or struct
 --> tests/ui/func/form.rs:8:32
  |
8 | pub async fn send_post(#[form] s: &String) -> Result<String> {}
  |                                ^^^^^^^^^^
